Undefined subroutine &GCMailProgramsDialog::localName called at /usr/bin/../lib/gcstar/GCMail.pm line 328.

diff -Nuar a/gcstar/lib/gcstar/GCMail.pm b/gcstar/lib/gcstar/GCMail.pm
--- a/gcstar/lib/gcstar/GCMail.pm	2021-03-04 11:24:09.000000000 +0100
+++ b/gcstar/lib/gcstar/GCMail.pm	2021-05-10 21:23:57.000000000 +0000
@@ -291,6 +291,7 @@
 
 {
     package GCMailProgramsDialog;
+    use GCUtils 'localName';
     use base 'GCModalDialog';
     use XML::Simple;
 
